src/components/Overview/AboutSection.tsx

```diff
@@ -0,0 +1,140 @@
+            <CardTitle>Become Continuous</CardTitle>
+            <CardBody>
+              You can run your containerized applications in the bundled, shared Kubernetes-based
+              environments that are powered by Red Hat OpenShift. You can share the routes created
+              for your services and you can continuously build, test and rollout your applications
+              with a simple `git push` or acceptance of a pull request.
+            </CardBody>
+          </Card>
+        </CardBody>
+      </Card>
+    </GridItem>
+    <GridItem span={4}>
+      <Stack hasGutter>
+        <StackItem>
+          <Card isLarge>
+            <CardTitle>Related technologies</CardTitle>
+            <CardBody>
+              <Split hasGutter>
+                <SplitItem>
+                  <img src={nodeLogo} alt="Node.js logo" width="30px" height="30px" />
+                </SplitItem>
+                <SplitItem isFilled>Node.js</SplitItem>
+              </Split>
+              <Split hasGutter>
+                <SplitItem>
+                  <img src={clairLogo} alt="Clair logo" width="30px" height="30px" />
+                </SplitItem>
+                <SplitItem isFilled>Clair</SplitItem>
+              </Split>
+              <Split hasGutter>
+                <SplitItem>
+                  <img src={pythonLogo} alt="Python logo" width="30px" height="30px" />
+                </SplitItem>
+                <SplitItem isFilled>Python</SplitItem>
+              </Split>
+              <Split hasGutter>
+                <SplitItem>
+                  <img src={springLogo} alt="Spring logo" width="30px" height="30px" />
+                </SplitItem>
+                <SplitItem isFilled>Spring</SplitItem>
+              </Split>
+              <Split hasGutter>
+                <SplitItem>
+                  <img src={tektonLogo} alt="Tekton logo" width="30px" height="30px" />
+                </SplitItem>
+                <SplitItem isFilled>Tekton</SplitItem>
+              </Split>
+              <Split hasGutter>
+                <SplitItem>
+                  <img
+                    src={openPolicyAgentLogo}
+                    alt="Open Policy Agent logo"
+                    width="30px"
+                    height="30px"
+                  />
+                </SplitItem>
+                <SplitItem isFilled>Open Policy Agent</SplitItem>
+              </Split>
+              <Split hasGutter>
+                <SplitItem>
+                  <img src={redhatLogo} alt="Red Hat logo" width="30px" height="30px" />
+                </SplitItem>
+                <SplitItem isFilled>OpenShift</SplitItem>
+              </Split>
+            </CardBody>
+          </Card>
+        </StackItem>
+        <StackItem>
+          <Card isLarge>
+            <CardTitle>Contact us</CardTitle>
+            <CardBody>
+              Join the{' '}
+              <a href="https://rhdevnation.slack.com/join/" target="_blank" rel="noreferrer">
```

Comment:
This link is redirecting to 404 page, maybe we should just redirect to `https://rhdevnation.slack.com`

Reply:
We have this link to create invites https://dn.dev/DevNationSlack

Reply:
Updated the link that takes user to join page.